src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -44,6 +49,14 @@
  * @since 0.7
  */
 public final class ManifestsTest {
+    // @checkstyle JavadocVariableCheck (8 lines)
+    public static final String TEST_MANIFEST = "test.mf";
+    public static final String TEST_MANIFEST_2 = "another-test.mf";
+    public static final String TEST_ATTRIBUTE = "some test attribute";
```

Comment:
@nesterov-n 
> Another typical mistake is to use constants in unit tests to avoid duplicate string/numeric literals in test methods. Don't do this! Every test method should work with its own set of input values.